src/Silk.Core/Commands/General/EnlargeCommand.cs

```diff
@@ -5,62 +5,63 @@
-        public async Task Enlarge(CommandContext ctx, DiscordEmoji emoji)
-        {
+	[Category(Categories.General)]
+	public class EnlargeCommand : BaseCommandModule
+	{
+		[Command("enlarge")] [Aliases("enbiggen", "emoji", "emote")]
+		[Description("Displays a larger version of the provided emoji or custom emote.")]
+		public async Task Enlarge(CommandContext ctx, DiscordEmoji emoji)
+		{
 
-            var embed = new DiscordEmbedBuilder().WithColor(new("2F3136"));
-            if (emoji.Id != 0) // Guild emote.
-            {
-                var message = new DiscordMessageBuilder();
-                embed.WithFooter(emoji.Name);
-                embed.WithImageUrl(emoji.Url + "?size=2048");
-                message.WithEmbed(embed)
-                    .WithReply(ctx.Message.Id);
-                await ctx.RespondAsync(message);
-            }
-            else // Unicode emote.
-            {
-                await ctx.TriggerTypingAsync();
+			DiscordEmbedBuilder? embed = new DiscordEmbedBuilder().WithColor(new("2F3136"));
+			if (emoji.Id != 0) // Guild emote.
+			{
+				var message = new DiscordMessageBuilder();
+				embed.WithFooter(emoji.Name);
+				embed.WithImageUrl(emoji.Url + "?size=2048");
+				message.WithEmbed(embed)
+					.WithReply(ctx.Message.Id);
+				await ctx.RespondAsync(message);
+			}
+			else // Unicode emote.
+			{
+				await ctx.TriggerTypingAsync();
 
-                embed.WithFooter(emoji.GetDiscordName().Replace(":", ""));
-                embed.WithImageUrl("attachment://emote.jpeg");
+				embed.WithFooter(emoji.GetDiscordName().Replace(":", ""));
+				embed.WithImageUrl("attachment://emote.jpeg");
 
-                var image = RenderEmoji(emoji.Name);
+				Stream? image = RenderEmoji(emoji.Name);
 
-                var message = new DiscordMessageBuilder()
-                    .WithEmbed(embed)
-                    .WithFile("emote.jpeg", image)
-                    .WithReply(ctx.Message.Id);
+				DiscordMessageBuilder? message = new DiscordMessageBuilder()
+					.WithEmbed(embed)
+					.WithFile("emote.jpeg", image)
+					.WithReply(ctx.Message.Id);
 
-                await message.SendAsync(ctx.Channel);
-            }
-        }
+				await message.SendAsync(ctx.Channel);
+			}
+		}
 
-        private Stream RenderEmoji(string unicodeEmoji)
-        {
+		private Stream RenderEmoji(string unicodeEmoji)
+		{
 
-            Stream svgStream,
-                imageStream = new MemoryStream();
+			Stream svgStream,
+				imageStream = new MemoryStream();
 
-            var emojiHex = char.ConvertToUtf32(unicodeEmoji, 0).ToString("X4");
-            var url = $"https://twemoji.maxcdn.com/2/svg/{emojiHex.ToLower()}.svg";
-            svgStream = new WebClient().OpenRead(url);
+			var emojiHex = char.ConvertToUtf32(unicodeEmoji, 0).ToString("X4");
+			var url = $"https://twemoji.maxcdn.com/2/svg/{emojiHex.ToLower()}.svg";
+			svgStream = new WebClient().OpenRead(url);
```

Comment:
Can use `HttpClient` instead of old `WebClient`:

```csharp
using var client = _httpClientFactory.CreateClient();
Stream svgStream = await client.GetStreamAsync(url);
```

Then make the return of the helper method return `Task<Stream>` , inject the `IHttpClientFactory` from the constructor, and await helper in calling method 🚀

Reply:
@Giggitybyte this was your doing

Reply:
your FUCKING mother

Reply:
Sounds about right